ERCS/erc-7529.md

```diff
@@ -0,0 +1,166 @@
+
+**Example 2**:
+
+A user visits nftmarketplace.io to buy a limited release NFT from theirfavoritebrand.com. The marketplace app can leverage this ERC to allow the user to search by domain name and also indicate to the user that an NFT of interest is indeed an authentic asset associated with theirfavoritebrand.com. 
+
+## Specification
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174.
+
+#### Definition: eTLD+1 
+
+The term TLD stands for *top-level domain* and is always the part of a domain name which follows the final dot in a URL (e.g. `.com` or `.net`). If only domains directly under TLDs where registrable by a single organization, then it would be guaranteed that `myexample.com`, `abc.myexample.com`, and `def.myexample.com` all belonged to the same organization. 
+
+However, this is not the case in general since many DNS registrars allow organization to register domain names below the top level (examples include `sussex.ac.uk` and `aber.ac.uk` which are controlled by different institutions). These types of domains are referred to as eTLDs (effective top-level domains) and represent a domain under which domain names can be registered by a single organization. For example, the eTLD of `myexample.com` is `.com` and the eTLD of `sussex.ac.uk` is `.ac.uk` since individual organizations can be issued their own domain names under both `.com` and `.ac.uk`. 
+
+Therefore, an eTLD+1 is an eTLD *plus* this next part on the domain name. Since eTLDs are by definition registerable, all domains with the same eTLD+1 are owned by the same organization, which makes them appropriate to utilize in this proposal for associating a smart contract with a single business or organization entity. 
+
+
+### Contract Pointers in TXT Records 
+
+The owner of an eTLD+1 domain name MUST create a TXT record in their DNS settings that serves as a pointer to all relevant smart contracts they wish to associate with their domain. 
+
+[TXT records](https://www.rfc-editor.org/rfc/rfc1035#section-3.3.14) are not intended (nor permitted by most DNS servers) to store large amounts of data. Every DNS provider has their own vendor-specific character limits. However, an EVM-compatible address string is 42 characters, so most DNS providers will allow for dozens of contract addresses to be stored under a single record. Furthermore, a domain is allowed to have multiple TXT records associated with the same host and the content of all duplicate records can be retrieved in a single DoH query. 
+
+A TXT record pointing to an organization's smart contracts MUST adhere to the following schema:
+
+- `HOST`: ERC-7529.<`chain_id`>._domaincontracts
+- `VALUE`: \<`address 1`\>,\<`address 2`\>,...
```

Comment:
So the host would be `ERC-7529.1._domaincontracts` for mainnet, and `ERC-7529.11155111._domaincontracts` for sepolia?

Can we format this like:

```suggestion
- `HOST`: `ERC-7529.<chain_id>._domaincontracts` (where `<chain_id>` is replaced by the decimal representation of the chain id)
- `VALUE`: \<`address 1`\>,\<`address 2`\>,...
```

Putting the ERC-7529 in backticks will make the linter shut up.

Reply:
done

Reply:
done